src/main.cpp

```diff
@@ -276,6 +292,32 @@ int main(int argc, char *argv[])
         channel->sendMessage(QByteArray("OPEN_SHADER\n") +
                              shaderName.c_str());
     }
+    if(hookSpecDef != "")
+    {
+        QByteArray msg;
+        QByteArray message = QByteArray("HOOK");
+        for(int i=0; i<hookPayload.size(); i++)
+            message = message + QByteArray("\n")
+                              + QByteArray(hookSpec[i].data(), (int)hookSpec[i].size()) + QByteArray("\n")
+                              + QByteArray(hookPayload[i].data(), (int)hookPayload[i].size());
```

Comment:
Style nitpick: for consistency, please insert some braces here.

If the body of the loop (of if statement) is a single line, I prefer to leave the braces off, but when it starts to cover multiple lines I think it's best to add some.  (I'm probably not entirely consistent with this myself...)